resources/public/api/conf/1.0/common/pathParameters.yaml

```diff
@@ -9,3 +9,23 @@ components:
       schema:
         type: string
         example: "a1e8057e-fbbc-47a8-a8b4-78d9f015c253"
+    businessId:
+      name: businessId
+      in: path
+      description: |
+        An identifier for the business, unique to the customer.<p>Must conform to the regular expression `^X[A-Z0-9]{1}IS[0-9]{11}$`
+      schema:
+        type: string
+        pattern: ^X[A-Z0-9]{1}IS[0-9]{11}$
+        example: XAIS12345678910
+      required: true
+
+    nino:
+      name: nino
+      in: path
+      description: |
+        National Insurance number, in the format `AA999999A`
+      schema:
+        type: string
+        example: TC663795B
+      required: true
```

Comment:
I didnt include it as all the other Nino path parameters didn't include it so I wanted to keep it consistent! Happy to add if you think its needed

Reply:
I don't know. There may be valid reasons not to include it, but I imagine that any time somebody needs to change the NINO validation on the front-end, they'd know to update this file too. So I don't see the harm in adding it.

I've seen a couple of (supposedly equivalent) NINO regexes around, so I wouldn't want to be the person on the other end guessing why my NINO is rejected, in case our NINO validation isn't consistent across all APIs.

Then again, anybody should be able to generate a valid NINO without much fuss because you gave an example. The problems might come when somebody tries to automate NINO generation without knowing the rules. I don't know how likely that is to happen because everything uses a NINO. They're not likely to come to this specific API for help in that regard.

We've got some APIs where the documented NINO regex is different from what the API does. If this had been one of them, I might not have suggested copying bad regexes into another doc. But this one doesn't appear to have the `^$` problems that some of the others do.

I leave the decision to you, but I'm happy to discuss this more.

Reply:
Added it!